dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.2/com.ibm.websphere.appserver.jaxb-2.2.feature

```diff
@@ -14,8 +14,11 @@ IBM-Process-Types: client, \
  server
 Subsystem-Name: Java XML Bindings 2.2
 -features=\
-  com.ibm.websphere.appserver.classloading-1.0,\
-  com.ibm.websphere.appserver.internal.jaxb-2.2
```

Comment:
Why did we lose the internal.jaxb-2.2 feature that always installed the bundle.  Is this not used by any other feature that always wants our jaxb bundles even on Java 8?

Reply:
I'm getting rid of the internal.jaxb-2.2 feature because it doesn't keep JAX-B internal (issue #4404 shows that JAX-B API still leaked out into the app classloader).  So the new approach will be this "optional and conditional" JAXB feature which only resolves on JDK 9+, but the public jaxb-2.2 feature needs to continue to resolve JAX-B regardless of JDK version.